### DISCUSSION TITLE
Can’t get data access

### DISCUSSION THREAD

**Discussion by Hannah-hue102** [now-i-know-my-abcd/docs#90] [Stats & Code Help]
How long has it taken others to gain approval after they submitted the data access request? The website says it takes 10 days but it has been 8 weeks. Just wondering if others had a similar experience?

**Comment by angielaird** [comment]
8 weeks is definitely too long, it sounds like your request got hung up somewhere. Send a message to the NDA Help Desk and see if they can help. If you have any other questions, feel free to DM/email me and I'm happy to help provide guidance. Good luck!

**Reply by Hannah-hue102** [reply]
Thank you, Angie! We just sent an email to the NDA help desk.
